### PR TITLE
Nmr 5244 use spinlab dataset library

### PR DESCRIPTION
**Pull request by vfiack**

### CODE REVIEW

File: nmrfx-bom/pom.xml [hunk 1]
```diff
@@ -19,6 +19,12 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>com.nanalysis</groupId>
+                <artifactId>spinlab-dataset</artifactId>
+                <version>1.1.0</version>
```
Comment (vfiack) [inline nmrfx-bom/pom.xml:25]: Repository for this project is here: https://github.com/nanalysis/spinlab-dataset

File: nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/rs2d/RS2DData.java [hunk 2]
```diff
@@ -235,29 +240,27 @@ private void openParFile(String parpath, boolean processed) throws IOException {
...
                 seriesDocument = readDocument(seriesPath);
             }
             groupDelay = readGroupDelay();
-            obsNucleus = header.getString(OBSERVED_NUCLEUS);
+            obsNucleus = header.get(OBSERVED_NUCLEUS).stringValue();
             Double obsFreq = null;
-            Double obsSW = header.getDouble(SPECTRAL_WIDTH);
-            tempK = header.getDouble(SAMPLE_TEMPERATURE);
-            String baseSFName = "";
+            double obsSW = header.<NumberValue>get(SPECTRAL_WIDTH).getValueAsHertz(header);
```
Comment (vfiack) [inline nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/rs2d/RS2DData.java:251]: This is the reason for the change: Spinlab data files can store SW as Hz or Ppm, reading the value directly from the xml is error-prone.

**Comment by vfiack** [comment]
Build failing because dependency compiled for java17.

**Review by JannalieT — COMMENTED** [review]
I tried loading a cosy from the data used for testing the jcamp data, and the axis labeling seems a bit off. For the processed spinlab datasets, it looks like values are off by a factor of 1e6 and the values for the unprocessed spinlabl datasets loaded and then processed, do not match the jcamp data at all.

cosy-spinlab

**Comment by vfiack** [comment]
Interesting. I'll have to check with the current version to see if I broke something or if the import was already flawed.

**Review by JannalieT — APPROVED** [review]
retested with COSY and a JRES, looks good

**Pull request merged**